dataactcore/scripts/loadHistoricalFabs.py

```diff
@@ -291,7 +423,7 @@ def set_active_rows(sess):
         "FROM published_award_financial_assistance GROUP BY afa_generated_unique) sub_pafa " +
         "ON pafa.modified_at = sub_pafa.modified_at AND " +
         "COALESCE(pafa.afa_generated_unique, '') = COALESCE(sub_pafa.afa_generated_unique, '') " +
-        "WHERE COALESCE(UPPER(pafa.correction_late_delete_ind), '') != 'D' AS selected " +
+        "WHERE COALESCE(UPPER(pafa.correction_late_delete_ind), '') != 'D') AS selected " +
```

Comment:
Why is there an extra parenthesis here? Wouldn't this break the SQL?

Reply:
No, mike accidentally deleted it when copy pasting before so we had to add it back in to work